query/query1_test.go

```diff
@@ -669,6 +669,23 @@ func TestHasFuncAtRootWithAfter(t *testing.T) {
 	require.JSONEq(t, `{"data": {"me":[{"friend":[{"count":1}],"name":"Rick Grimes","uid":"0x17"},{"friend":[{"count":1}],"name":"Andrea","uid":"0x1f"}]}}`, js)
 }
 
+func TestHasFuncAtRootWithAfterOnUIDs(t *testing.T) {
+
+	query := `
+	{
+		var(func: has(name)) {
+			uids as uid
+		}
+		me(func: uid(uids), first: 2, after: 0x5) {
```

Comment:
This is looking good, one thing to verify is that if it also works at a nested level. So if the second query was something like

```
{
  me(func: uid(1, 2, 3, 4, 5) {
    friend @filter(uid(uids), after: ...) {
      name
    }
  }
}
```